xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/main/java/org/xwiki/refactoring/internal/DefaultLinkRefactoring.java

```diff
@@ -87,7 +86,7 @@
     private ContentParser contentParser;
 
     @Inject
-    private ReferenceRenamer renamer;
+    private DefaultReferenceRenamer renamer;
```

Comment:
Out of curiosity, why do you inject component instead of a role?

Reply:
answer: it has to be a bug ;) Normally our build should catch this.

Reply:
Yes it's a bug